cypress/e2e/data-import/e2e/mrc-import-file-for-update-items.cy.js

```diff
@@ -280,8 +280,10 @@ describe('ui-data-import', () => {
   // create Field mapping profile for export
   const exportMappingProfile = {
     name: `autoTestMappingProf.${getRandomPostfix()}`,
+    holdingsTransformation: 'Holdings - HRID',
     holdingsMarcField: '901',
     subfieldForHoldings:'$a',
+    itemTransformation: 'Item - HRID',
```

Comment:
let's move this string to const variables to separate constants file which is located in the support directory

Reply:
Done